lib/game/doodle_dash.dart

```diff
@@ -14,10 +14,9 @@ class DoodleDash extends FlameGame
 
   final Player dash = Player();
   final World _world = World();
-  PlatformManager platformManager =
-      PlatformManager(maxVerticalDistanceToNextPlatform: 1000);
+  PlatformManager platformManager = PlatformManager();
 
-  int screenBufferSpace = 100;
+  int screenBufferSpace = 200;
```

Comment:
Not sure this is actually necessary. I changed it last week and can't remember why.